src/components/aside/Aside.jsx

```diff
@@ -6,11 +6,11 @@ import {
   filterRatingData,
 } from "../../data/filter-data/filter.data";
 import "./Aside.css";
-export function Aside() {
+export function Aside({ asideStyles }) {
   const { dispatch, state } = useFilter();
 
   return (
-    <aside className="aside-product">
+    <aside className="aside-product" style={{ display: asideStyles }}>
```

Comment:
avoid using inline style

Reply:
i used this to show filter in mobile view